### ISSUE TITLE
savefig() does not hold for title, label, xlog, etc parameters.

### ISSUE THREAD

**Issue by carlomontec** [jheinen/GR.jl#287]
Hi,

I cant save the plots to PDF or PNG accounting for additional parameters of the plot:

```
 GR.plot(rand(50,3),  xlabel="Q_Y", ylabel="Q_Z", title="Rel. error ", size=(1600,800))

 GR.savefig("test.png")

```
In the gksqt window (Linux) I can see how the plot is regenerated with defaults, looking the same as the saved file. I assume is related to #276 ? 

Thank you for your great work.

**Comment by heliosdrm** [comment]
Yes, the high-level API of GR is being revised. Hopefully it will be ready soon, but if you can't wait, for the time being you may use the experimental version that is packaged as [GRUtils](https://github.com/heliosdrm/GRUtils.jl). This particular issue is already fixed there.

**Comment by heliosdrm** [comment]
Maybe this was closed by de79425d (released in GR v0.46.0)?
